### PR TITLE
Cleaned up samples and fixed style.

### PR DESCRIPTION
**Pull request by vqtran**
Deleted a bunch of unnecessary stuff. Fixed errors, and fixed Java
style.

### CODE REVIEW

File: src/main/java/speakql/SpeakQLSpeechlet.java [hunk 1]
```diff
@@ -43,375 +34,410 @@
...
+      throw new SpeechletException("Invalid Intent");
     }
-
-    @Override
-    public SpeechletResponse onLaunch(final LaunchRequest request, final Session session)
-            throws SpeechletException {
-        log.info("onLaunch requestId={}, sessionId={}", request.getRequestId(),
-                session.getSessionId());
-
-        return getWelcomeResponse();
```
Comment (GabeLoins) [inline src/main/java/speakql/SpeakQLSpeechlet.java]: why did you delete this?

File: src/main/java/speakql/SpeakQLSpeechlet.java [hunk 2]
```diff
@@ -43,375 +34,410 @@
...
+   * Download JSON-formatted list of events from Wikipedia, for a defined
+   * day/date, and return a String array of the events, with each event
+   * representing an element in the array.
+   * 
+   * @param month The month to get events for, example: April
+   * @param date The date to get events for, example: 7
+   * @return String array of events for that date, 1 event per element of the
+   *     array
+   */
+  private ArrayList<String> getJsonEventsFromWikipedia(
```
Comment (GabeLoins) [inline src/main/java/speakql/SpeakQLSpeechlet.java]: why did you keep this function?

File: src/main/java/speakql/SpeakQLSpeechlet.java [hunk 3]
```diff
@@ -43,375 +34,410 @@
...
+   * list of events and split the events into a String array of individual
+   * events. Run Regex matchers to make the list pretty by adding a comma after
+   * the year to add a pause, and by removing a unicode char.
+   * 
+   * @param text The JSON formatted list of events/births/deaths for a certain
+   *     date.
+   * @return String array of events for that date, 1 event per element of the
+   *     array.
+   */
+  private ArrayList<String> parseJson(String text) {
```
Comment (GabeLoins) [inline src/main/java/speakql/SpeakQLSpeechlet.java]: why did you keep this function

File: src/main/java/speakql/SpeakQLSpeechlet.java (hunk 1 above)
Comment (vqtran) [inline src/main/java/speakql/SpeakQLSpeechlet.java]: its still there i just changed the beginning of this file to include variables that were deleted for some reason and were throwing errors because of it

File: src/main/java/speakql/SpeakQLSpeechlet.java (hunk 3 above)
Comment (vqtran) [inline src/main/java/speakql/SpeakQLSpeechlet.java]: I didn't prune the logic here yet, I assumed that you kept HistoryBuff stuff for a reason

File: src/main/java/speakql/SpeakQLSpeechlet.java (hunk 2 above)
Comment (vqtran) [inline src/main/java/speakql/SpeakQLSpeechlet.java]: Same as above

**Comment by GabeLoins** [comment]
SHIP IT
